### DISCUSSION TITLE
Micropython-stubber: Proposed change to stub naming convention

### DISCUSSION THREAD

**Discussion by Josverl** [BradenM/micropy-cli#281] [Ideas]
@BradenM ,

Apologies for barging in here , but I wanted to ask your and others feedback on a change in naming convention that may affect `micropy-cli` and its users.

I have started a discussion in the general stubs repo : https://github.com/Josverl/micropython-stubs/discussions/107 

Simply stated , the idea is to move the version number of the stubs to a place consistent with the other types of stubs. 
current | `micropython-esp32-1_17`/*.pyi 
 ---| ----
**proposed** | **`micropython-1_17-esp32/*.pyi`** 

Please let me know in the [stubs repo](https://github.com/Josverl/micropython-stubs/discussions/107) what makes sense, and what not.

/Jos

**Comment by BradenM** [comment]
@Josverl,

Nonsense, you're welcome anytime :smile: 

Just followed up with a reply to your initial discussion. Thanks.
